typings/index.d.ts

```diff
@@ -0,0 +1,1909 @@
+/**
+ * Type definitions for "material-ui" v1.0.0-beta.2
+ * Project: https://github.com/callemall/material-ui/
+ * TypeScript Version: 2.4
+ */
+
+///<reference types="react" />
+///<reference types="enzyme" />
+
+declare namespace MaterialUI {
```

Comment:
namespaces aren't easy to work with. I suggest using es6 module system. Typescript is good at understanding exports.